src/Common/tests/XunitTraitsDiscoverers/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -23,7 +23,7 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
         public IEnumerable<KeyValuePair<string, string>> GetTraits(IAttributeInfo traitAttribute)
         {
             IEnumerable<object> ctorArgs = traitAttribute.GetConstructorArguments();
-            Contract.Assert(ctorArgs.Count() >= 2);
+            Debug.Assert(ctorArgs.Count() >= 2);
```

Comment:
Even if that's true, it'll fail fast with a message that includes all of the same information.  Plus, this should never happen, right?  If we actually expect it to ever happen, it shouldn't be either kind of assert and instead runtime checks that fall back to doing something reasonable.